twine-cert.txt

```diff
@@ -3256,3 +3256,236 @@
 [infschg_more_info_link]
     de = #Mehr Informationen::https://www.digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen#
     en = #More information::https://www.digitaler-impfnachweis-app.de/en/faq/#current-most-frequently-asked-questions#
+
+[infschg_popup_choose_federal_state_title]
+    de = Wählen Sie Ihr Bundesland
+    en = Choose your federal state
+    
+[infschg_popup_choose_federal_state_copy_1]
+    de = Die Regeln zur Maskenpflicht können je nach Bundesland variieren. Wählen Sie Ihr Bundesland aus, um Ihren aktuellen Maskenstatus sehen zu können.*
+    en = Mask requirement rules may vary by federal state. Select your federal state to see your current mask status.*
+    
+[infschg_popup_choose_federal_state_tag]
+    de = Bundesland
+    en = Federal state
+    
+[infschg_popup_choose_federal_state_box_content]
+    de = %d
+    en = %d
+    
+[infschg_popup_choose_federal_state_copy_2]
+    de = *Sie dürfen jederzeit das Bundesland von der entsprechenden Auswahl in den Einstellungen ändern.
+    en = *You may change the federal state from the corresponding selection in the settings at any time.
+    
+[infschg_popup_choose_federal_state_button]
+    ref = OK
+    
+[infschg_start_screen_status_green]
+    de = Keine Maskenpflicht
+    en = No mask required
+    
+[infschg_start_screen_status_blue]
+    de = Maskenpflicht
+    en = Mask required
```

Comment:
In general it would be more like a `mask mandate`

Reply:
"Mask mandatory"